Emma/emma_libs/memoryManager.py

```diff
@@ -28,7 +30,18 @@
 import Emma.emma_libs.mapfileProcessorFactory
 import Emma.emma_libs.memoryMap
 import Emma.emma_libs.categorisation
-import svgwrite
+
+
+class Element(IntEnum):
+    AddressStart = 0
+    AddressEnd = 1
+    AddressLength = 2
+    Fqn = 3
+    OriginalAddressStart = 4
+
+class PlottedElement(IntEnum):
+    AddressEnd = 0
```

Comment:
Why is the first character a majuscule?
Either ALL_CAPS or like normal variables (first character is a minuscule).
Choose your destiny ;-)